scripts/publish.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+source ${OKTA_HOME}/${REPO}/scripts/setup.sh
+
+REGISTRY="${ARTIFACTORY_URL}/api/npm/npm-okta"
+
+npm install -g @okta/ci-update-package
+npm install -g @okta/ci-pkginfo
```

Comment:
those will auto handle version bump iirc. I thought you'd like to manually maintain version?